DFKPS/DFKPS-IOC-01App/Db/DFKPS_common.db

```diff
@@ -133,3 +133,14 @@ record(bo, "$(P)DISABLE")
   field(VAL, "$(DISABLE=0)")
 }
 
+record(calcout, "$(P)CURR")
+{
+field(DESC, "Convert Raw Val to Current")
+field(CALC, "A * $(FRI)")
```

Comment:
Should this be "A / $(FWI)"? If not you need to define FRI in the st.cmd.

Sorry I didn't catch this in the last review.